src/clib/pio_nc.c

```diff
@@ -1340,8 +1340,8 @@ int PIOc_inq_attname(int ncid, int varid, int attnum, char *name)
     }
 
     if(ierr != PIO_NOERR){
-        LOG((1, "nc*_inq failed, ierr = %d", ierr));
-        return pio_err(ios, file, ierr, __FILE__, __LINE__);
+        LOG((1, "nc*_inq_attname failed, ierr = %d", ierr));
```

Comment:
Do you think this case should be a fatal error? The user is already specifying the attnum and inquiring the corresponding attribute name.

Reply:
The old code considered it as a "soft error" but maybe this case should be fatal

Reply:
I will add the change to convert nc_inq_attname from soft to
a fatal error as a separate commit.